src/ChatWindow.test.js

```diff
@@ -130,3 +131,25 @@ it("handles gracefully when the input form is submitted with a null or empty req
   // Nothing about the state should have changed.
   expect(finalState).toEqual(originalState);
```

Comment:
Verify that preventDefault() is being called.

Reply:
Verify preventDefault is being called in other tests as well wherever necessary.

Also, could you make preventDefaultSpy a class variable instead?

Reply:
Done.